scipy/stats/_stats_py.py

```diff
@@ -10559,24 +10559,14 @@ def lmoment(sample, order=None, *, axis=0, sorted=False, standardize=True):
                                      extra_field_names=['intercept_stderr'])
 
 
-def linregress(x, y=None, alternative='two-sided'):
+def linregress(x, y, alternative='two-sided'):
```

Comment:
Seems we missed our chance to make this function kwarg-only with 1.14..., c.f. https://github.com/scipy/scipy/issues/18703

Reply:
could do it now?